src/pages/contributors/jesse-glick.adoc

```diff
@@ -0,0 +1,107 @@
+= Jesse Glick
+:page-name: Jesse Glick
+:page-linkedin:
+:page-twitter: 
+:page-github: jglick
+:page-email:
+:page-image: avatar/jesse-glick.jpeg
+:page-pronouns:
+:page-location: Raleigh, North Carolina, USA
+:page-firstcommit: 2009
+:page-datepublished: 2024-11-05
+:page-featured: true
+:page-intro: Jesse Glick is a software engineer, currently residing in Raleigh, North Carolina, and has been involved with the Jenkins project for over a decade. Prior to contributing to Jenkins, Jesse worked as an engineer at Sun Microsystems. After incorporating Jenkins into his work at Sun, albeit in a small way, there wasn't much more to do with the project. However, that all changed when Kohsuke reached out to Jesse to find out if he would be interested in working on Jenkins full-time. Since joining CloudBees in 2012, Jesse has been an integral part of Pipeline development, and incorporating many functions that are now fundamental to current Pipeline operation. Jesse works constantly on improving the Pipeline experience, and shares his insights on how to improve not only the Jenkins project but the open-source workflow as a whole.
```

Comment:
```suggestion
:page-intro: Jesse Glick is a software engineer, currently residing near Raleigh, North Carolina, and has been involved with the Jenkins project for over a decade. Prior to contributing to Jenkins, Jesse worked as an engineer at Sun Microsystems. After incorporating Jenkins into his work at Sun, albeit in a small way, there wasn't much more to do with the project. However, that all changed when Kohsuke reached out to Jesse to find out if he would be interested in working on Jenkins full-time. Since joining CloudBees in 2012, Jesse has been an integral part of Pipeline development, and incorporating many functions that are now fundamental to current Pipeline operation. Jesse works constantly on improving the Pipeline experience, and shares his insights on how to improve not only the Jenkins project but the open-source workflow as a whole.
```

BTW https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line makes reviews in GH a lot more pleasant.

Reply:
@jglick I definitely agree with the one sentence per line, and that's typically what we use. The issue with doing that for the page intro is that it breaks it up completely and doesn't render as desired on the overview page. The goal for the intro is to force an `...` within the feature box to indicate there is more to read. Doing one sentence per line would make it so only the first line included in the tag is displayed and would look inadequate.
<img width="577" alt="Screenshot 2024-10-30 at 12 54 46 PM" src="https://github.com/user-attachments/assets/1f94fb74-5ab1-4a76-9eab-616570d45194">